src/pylon_camera/pylon_camera_parameter.cpp

```diff
@@ -81,6 +82,8 @@ void PylonCameraParameter::readFromRosParameterServer(const ros::NodeHandle& nh)
         nh.getParam("camera_info_url", camera_info_url_);
     }
 
+    nh.param("pixel_encoding", pixel_encoding_, 2);
```

Comment:
Please don't use a magic number (like '2') in the code if you are referring to an enum.